src/lib/serviceRoutines/postContextEntitiesByEntityId.cpp

```diff
@@ -51,12 +54,11 @@ std::string postContextEntitiesByEntityId
 )
 {
   std::string  entityId = (compV[0] == "v1")? compV[3] : compV[2];
```

Comment:
"v1" is fine by the moment. Not sure when "v2" will be born. Note that at the current moment is not corfirmed (or corfirmed) as a requirement to mantatin both "v1" and "v2" in the same piece of code, so let's leave this part as it is now.

NTC